src/components/ContextMenuItem.tsx

```diff
@@ -101,23 +92,12 @@ function ContextMenuItem({onPress, successIcon, successText, icon, text, isMini,
             style={StyleUtils.getContextMenuItemStyles(windowWidth)}
             isAnonymousAction={isAnonymousAction}
             focused={isFocused}
-            interactive={isThrottledButtonActive}
+            interactive
```

Comment:
Why it was changed?

Reply:
If you take a look at this part of **MenuItem.tsx**:

```javascript
type ResponsiveProps = {
    /** Function to fire when component is pressed */
    onPress: (event: GestureResponderEvent | KeyboardEvent) => void;

    interactive?: true;
};

type UnresponsiveProps = {
    onPress?: undefined;

    /** Whether the menu item should be interactive at all */
    interactive: false;
};
```

you can see that **interactive** depends on **onPress**. How do you think we should handle it?

Reply:
@JKobrynski I'm not sure if the typing is correct, I see other places in the code where `onPress` event is passed to the MenuItem, but  `interactive` can be both `true` or `false`. Example:
[one](https://github.com/VickyStash/Expensify/blob/main/src/pages/workspace/WorkspaceInitialPage.js#L315)
[two](https://github.com/VickyStash/Expensify/blob/main/src/components/MoneyTemporaryForRefactorRequestConfirmationList.js#L605)
[three](https://github.com/VickyStash/Expensify/blob/main/src/components/ReportActionItem/TaskView.js#L177)

Reply:
These are JS files, I think TypeScript will complain when they are migrated 😄 If I try to keep this property in it's original form, I get the following error:

```
 Types of property 'interactive' are incompatible.
    Type 'boolean' is not assignable to type 'true'.
```

Reply:
Yeah, but I guess we should rather update `MenuItem` typing then update the props we pass to the component.
See what kind of updates where made in [another PR](https://github.com/Expensify/App/pull/33645/files#diff-9db0c2baf85f38b9a238d03f1089a686039ab6c6b94ef412bda920293909b7a4L36) to escape adding extra `interactive` prop, I think you can do the same. What do you think?

Reply:
Thanks for linking that PR, very useful! PR updated 😄